src/actions/copy-async.ts

```diff
@@ -149,11 +148,10 @@ export async function copyAsync(
   ]);
 }
 
-export type CopySingleAsyncOptions = AppendOptions &
-  CopySingleOptions & {
-    append?: boolean;
-    processFile?: (this: MemFsEditor, filepath: string) => string | Promise<string | Buffer>;
-  };
```

Comment:
I don't know why this was there before... I can't see where those options are used. So I think it's safe to delete.